### DISCUSSION TITLE
some levels i've done

### DISCUSSION THREAD

**Discussion by Droooga** [Neverball/neverball#285] [Show and tell]
![image](https://user-images.githubusercontent.com/85588460/155160529-21d522e2-41fa-4d90-867c-b2f9341b80d7.png)
![image](https://user-images.githubusercontent.com/85588460/155160541-1ef2c491-b363-47bd-965e-cdccea78ee12.png)
![image](https://user-images.githubusercontent.com/85588460/155160554-669b812e-2c4a-4439-ac24-6bdad457d486.png)
![image](https://user-images.githubusercontent.com/85588460/155160575-7e4e1046-00d8-4740-9690-6629a41adfeb.png)
if you wanna play the levels: https://cdn.discordapp.com/attachments/334837133573488642/944060475258314792/dermokerset1.zip
the full sets gonna have 20 levels, but since my motivation keeps failing me it's gonna take a long time to do it

**Comment by parasti** [comment]
Cool!

**Comment by DonAlgo** [comment]
I loved this levelset! My favorite levels were the second and the last one! Keep going!
